src/java/org/apache/nutch/parse/ParseOutputFormat.java

```diff
@@ -338,6 +340,9 @@ public static String filterNormalize(String fromUrl, String toUrl,
       } catch (MalformedURLException e1) {
         return null; // skip it
       }
+
+
+
       if ("bydomain".equalsIgnoreCase(ignoreExternalLinksMode)) {
         String toDomain = URLUtil.getDomainName(targetURL).toLowerCase();
         if (toDomain == null || !toDomain.equals(origin)) {
```

Comment:
I did not have a chance to test `db.ignore.external.links.mode : byDomain` feature.

However, if this new feature should also be covered for overriding, then I am ready to do so.